docs/source/workflow.rst

```diff
@@ -111,11 +111,43 @@ A yelpsoa-configs master runs `generate_deployments_for_service <generated/paast
 frequently. The generated ``deployments.json`` appears in ``/nail/etc/services/service_name`` throughout the cluster.
 
 Marathon masters run `deploy_marathon_services <deploy_marathon_services.html>`_,
-a thin wrapper around `setup_marathon_job <setup_marathon_job.html>`_.
+a thin wrapper around ``setup_marathon_job``.
 These scripts parse ``deployments.json`` and the current cluster state,
 then issue comands to Marathon to put the cluster into the right state
 -- cluster X should be running version Y of service Z.
 
+``setup_marathon_job`` iterates over a list of service.instances to avoid
```

Comment:
This documentation is too low level to be in the "workflow" section.

Users don't really need to worry about these internals.

Please make a new section in the bounce docs for these and move them there. Day-to-day users don't need to know about these details